diff --git a/webrtc/api/api.gyp b/webrtc/api/api.gyp
index f63e7e4..ae98b1e 100644
--- a/webrtc/api/api.gyp
+++ b/webrtc/api/api.gyp
@@ -8,6 +8,9 @@
 
 {
   'includes': [ '../build/common.gypi', ],
+  'variables': {
+      'libjingle_so_name%': 'jingle'
+  },
   'conditions': [
     ['os_posix == 1 and OS != "mac" and OS != "ios"', {
       'conditions': [
@@ -22,6 +25,37 @@
         }],
       ],
     }],
+    ['OS=="linux"', {
+      'targets': [
+        {
+          'target_name': 'libjingle_so',
+          'product_name': '<(libjingle_so_name)',
+          'type': 'shared_library',
+          'lib': 'libjingle.so',
+          'dependencies': [
+            '<(webrtc_root)/system_wrappers/system_wrappers.gyp:field_trial_default',
+            '<(webrtc_root)/system_wrappers/system_wrappers.gyp:metrics_default',
+            'libjingle_peerconnection',
+          ],
+          'sources': [
+            'media/base/audioframe.h',
+          ],
+          'conditions': [
+            ['OS=="linux"', {
+              'defines': [
+                'HAVE_GTK',
+              ],
+              'link_settings': {
+                'libraries': [
+                  '<!@(pkg-config --libs-only-l gobject-2.0 gthread-2.0 gtk+-2.0)',
+                  '-lstdc++',
+                ],
+              },
+            }],
+          ],
+        }
+      ]
+    }],
     ['OS=="android"', {
       'targets': [
         {
